frame/referenda/src/benchmarking.rs

```diff
@@ -264,6 +264,19 @@ benchmarks_instance_pallet! {
 		assert_matches!(ReferendumInfoFor::<T, I>::get(index), Some(ReferendumInfo::Cancelled(_, _, None)));
 	}
 
+	refund_submission_deposit {
+		let (origin, index) = create_referendum::<T, I>();
+		let caller = frame_system::ensure_signed(origin.clone()).unwrap();
+		let balance = T::Currency::free_balance(&caller);
+		assert_ok!(Referenda::<T, I>::cancel(T::CancelOrigin::successful_origin(), index));
+		assert_matches!(ReferendumInfoFor::<T, I>::get(index), Some(ReferendumInfo::Cancelled(_, Some(_), _)));
+	}: _<T::RuntimeOrigin>(origin, index)
+	verify {
+		assert_matches!(ReferendumInfoFor::<T, I>::get(index), Some(ReferendumInfo::Cancelled(_, None, _)));
+		let new_balance = T::Currency::free_balance(&caller);
+		assert!(new_balance > balance);
```

Comment:
not sure why the generated weights contain only one read and write of ReferendumInfoFor, and nothing from unreserve

Reply:
@ggwpez may be you know?

Reply:
I think that account is already touched since we deduce the transaction fees - so its whitelisted.  
But I cannot find the code right now…

Reply:
yup - pretty sure this is true.